cel.md

```diff
@@ -0,0 +1,13 @@
+## Setup
+
+For more details about the CEL input settings, check the [Filebeat documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-cel.html).
+
+### Collecting logs from CEL
+
+To collect logs via CEL, select **Collect logs via CEL** and configure the following parameters:
+
+- Dataset name: Dataset to write data to. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).
+- OAuth2 Client ID: Client ID used for OAuth2 authentication
+- OAuth2 Client Secret: Client secret used for OAuth2 authentication
+- OAuth2 Token URL: The URL endpoint that will be used to generate the tokens during the oAuth2 flow. It is required if no oauth_custom variable is set or provider is not specified in oauth_custom variable.
```

Comment:
Is the idea to add only `required: true` options to this section? Just wondering why we would add `OAuth2` options here, but not inside `httpjson.md`?
I think asking users to configure [`program`](https://github.com/elastic/integrations/blob/main/packages/cel/manifest.yml#L59-L79) might be useful. @efd6 WDYT?

Reply:
I'll push the changes up shortly, but I tried to rework the docs here

The idea is to add the common settings that need configuring for integrations that use the CEL input so we can include this doc snippet automatically for those integrations. 

We want it helpful, but generic enough that it won't be incorrect across different integrations doing things slighly differently.